packages/nodes-base/nodes/Tributech/descriptions/twin-api/RelationshipsDescription.ts

```diff
@@ -0,0 +1,152 @@
+import {
+	INodeProperties,
+} from 'n8n-workflow';
+
+export const relationshipsOperations = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		description: 'The operation that should be executed',
+		type: 'options',
+		displayOptions: {
+			show: {
+				resource: [
+					'relationships',
+				],
+			},
+		},
+		options: [
+			{
+				name: 'Create Relationship',
+				description: 'Create Relationship',
+				value: 'postRelationship',
+			},
+			{
+				name: 'Delete Relationship By Id',
+				description: 'Delete Relationship By Id',
+				value: 'deleteRelationshipById',
+			},
+			{
+				name: 'Get All Relationships',
+				description: 'Get All Relationships',
+				value: 'getAllRelationships',
+			},
+			{
+				name: 'Get Incoming Relationships',
+				description: 'Get Incoming Relationships',
+				value: 'getIncomingRelationships',
+			},
+			{
+				name: 'Get Outgoing Relationships',
+				description: 'Get Outgoing Relationships',
+				value: 'getOutgoingRelationships',
+			},
+			{
+				name: 'Get Relationship By Id',
+				description: 'Get Relationship By Id',
+				value: 'getRelationshipById',
+			},
+			{
+				name: 'Update Relationship',
+				description: 'Update Relationship',
+				value: 'updateRelationship',
+			},
+		],
+		default: 'getOutgoingRelationships',
+	},
+] as INodeProperties[];
+
+export const relationshipsFields = [
+	{
+		displayName: 'Additional Fields',
+		name: 'additionalFields',
+		type: 'collection',
+		placeholder: 'Add Field',
+		default: {},
+		displayOptions: {
+			show: {
+				resource: [
+					'relationships',
+				],
+				operation: [
+					'getAllRelationships',
+				],
+			},
+		},
+		options: [
```

Comment:
Same as above, related to `Get all` operations. We can remove page number and page size and fetch a single page by default and a `Return all` toggle that makes the node automatically iterate over all pages and return all data in a single execution.